csharp/src/DropboxContentHasher.cs

```diff
@@ -20,7 +20,7 @@
 ///     while (true)
 ///     {
 ///         int n = file.Read(buf, 0, buf.Length);
-///         if (n &lt;= 0) break;  // EOF
+///         if (n <= 0) break;  // EOF
```

Comment:
Because the comment uses XML, I think have to escape angle brackets.  Is that not the case?